blueprints/addon/files/.travis.yml

```diff
@@ -38,8 +38,6 @@ jobs:
     - stage: "Tests"
       name: "Tests"
       script:
-        - <% if (yarn) { %>yarn<% } else { %>npm run<% } %> lint:hbs
-        - <% if (yarn) { %>yarn<% } else { %>npm run<% } %> lint:js
         - <% if (yarn) { %>yarn<% } else { %>npm<% } %> test
```

Comment:
Is this going to run `test:ember-try`, followed by the same thing below?

Reply:
Ya, but you are right (at least this is what I think you are saying): we want this to run the “basic tests” not “additional tests”. I’ll update.

Reply:
Updated, thanks for catching this!